src/hooks/create-use-cache-call.js

```diff
@@ -0,0 +1,35 @@
+import { useDrizzleState } from '.'
+
+export default drizzle => (
+  contractNameOrContractNames,
+  methodNameOrFunction,
+  ...args
+) => {
+  const isFunction = typeof methodNameOrFunction === 'function'
+  const drizzleState = useDrizzleState(drizzleState => {
+    if (isFunction) {
+      return contractNameOrContractNames.reduce((acc, contractName) => {
```

Comment:
Could this be refactored to the following?

```js
reduce((acc, contractName) => ({
  ...acc,
  [contractName]: drizzleState.contracts[contractName]
}), {})
```

Reply:
Sorry, that's the optimizer in me. Changed.